src/native/util/common.h

```diff
@@ -34,17 +35,34 @@ namespace noobaa
 
 #define LOG(x) std::cerr << LOG_PREFIX() << x << std::endl
 
+#define SYS_LOG(x)                                      \
+do {                                                    \
+        std::ostringstream oss;                         \
+        oss << "" << LOG_PREFIX() << x << std::endl;    \
+        std::string log_str = oss.str();                \
+        const char* log_msg = log_str.c_str();          \
+        int facility = LOG_LOCAL0;                      \
+        int priority = 0;                               \
+        ::syslog(priority | facility, "%s", log_msg);   \
+    } while (0)
 // to use DBG the module/file should use either DBG_INIT or DBG_INIT_VAR.
 #define DBG_INIT(level) static int __module_debug_var__ = level
 #define DBG_INIT_VAR(debug_var) static int& __module_debug_var__ = debug_var
 #define DBG_SET_LEVEL(level) __module_debug_var__ = level
 #define DBG_GET_LEVEL() (__module_debug_var__)
 #define DBG_VISIBLE(level) (level <= __module_debug_var__)
-#define DBG(level, x)                        \
-    do {                                     \
-        if (DBG_VISIBLE(level)) {            \
-            LOG("[L" << level << "] " << x); \
-        }                                    \
+#define DBG(level, x)                                                   \
+    do {                                                                \
+        if (DBG_VISIBLE(level)) {                                       \
+            const char* syslog_enabled = std::getenv("LOG_TO_SYSLOG_ENABLED"); \
+            const char* stderr_enabled = std::getenv("LOG_TO_STDERR_ENABLED"); \
```

Comment:
OK on second thought I think this is too excessive - getenv is doing linear search over all the variables and calling it twice per debug log print will be bad for performance. lets figure out how to push this values down from js on demand - for example with fs_napi set_debug_level() or maybe better just add a new method.